### DISCUSSION TITLE
Working on subsets of stations

### DISCUSSION THREAD

**Discussion by seismolab-uct** [ROBelgium/MSNoise#381] [Q&A] [answered]
Hi I hope this is the right place to ask this. I am interested in using only a subset of stations from my database, is there a way of specifying this in the configuration or via another method? I currently have 202 stations but I would like to work on roughly 30 stations only at first. Thanks for any suggestions

**Comment by ThomasLecocq** [comment, marked as answer]
you can define stations' "USED" parameter to True or False, which then includes them in the steps. Do that before running the "new_jobs" step, so that only "used" stations will be taken into account for the jobs.

**Reply by seismolab-uct** [reply]
oh right, I completely missed that, that's embarrassing. 
Thank you Thomas

**Reply by ThomasLecocq** [reply]
No worries! It's a bit more complicated if you'd already had computed the whole workflow and needed to filter the dvvs !
